fastly/backend_test.go

```diff
@@ -21,10 +21,10 @@ func TestClient_Backends(t *testing.T) {
 			ServiceVersion: tv.Number,
 			Name:           "test-backend",
 			Address:        "integ-test.go-fastly.com",
-			Port:           Uint(1234),
 			ConnectTimeout: Uint(1500),
 			OverrideHost:   "origin.example.com",
 			SSLCiphers:     "DHE-RSA-AES256-SHA:DHE-RSA-CAMELLIA256-SHA:AES256-GCM-SHA384",
+			SSLSNIHostname: "ssl-hostname.com",
```

Comment:
ℹ️ I've added a test assertion for `SSLSNIHostname` as that was also mentioned as being problematic on the Fastly customer support ticket.